org.opent2t.sample.hub.superpopular/com.nest.hub/js/common.js

```diff
@@ -1,20 +1,13 @@
-/* jshint esversion: 6 */
-/* jshint node: true */
-
 'use strict';
 
 class accessTokenInfo {
         constructor(
             accessToken,
-            refreshToken,
-            tokenType, // ex: 'bearer'
-            scopes // ex: 'full_access'
+            ttl // time to live
```

Comment:
>ttl // time to live [](start = 12, length = 19)

I'm going to be coming in an re-doing some of our access token work in the next couple days so that we can support an arbitrary number of tokens and expirations, so this is fine as is, but will be changing.